pyflakes/checker.py

```diff
@@ -97,7 +97,7 @@ def iter_child_nodes(node, omit=None, _fields_order=_FieldsOrder()):
     are nodes and all items of fields that are lists of nodes.
     """
     for name in _fields_order[node.__class__]:
-        if name == omit:
+        if omit and name in omit:
```

Comment:
You need to support the old calling signature as well as the new one.

Reply:
hmm, it is, but only loosely because `'body' in 'body'` is True.  Anyway.. it works.
Add a note to the docstring that `omit` can be a single string or list of strings.